test/WebJobs.Script.Tests.Integration/BlobLeaseManagerTests.cs

```diff
@@ -119,76 +130,6 @@ public class BlobLeaseManagerTests
         }
 
         [Fact]
-        public void AcquiringLease_WithServerError_LogsAndRetries()
```

Comment:
These tests are testing cases that are now handled by the SDK.  

---
In reply to: [123674215](https://github.com/Azure/azure-webjobs-sdk-script/pull/1615#discussion_r123674215) [](ancestors = 123674215)